### ISSUE TITLE
DeprecationWarning: Buffer() is deprecated due to security and usability issues.

### ISSUE THREAD

**Issue by iraycd** [rezeus/korauter#1]
```javascript
yarn add v1.7.0
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
⠂ @rezeus/korauter(node:6559) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
[3/5] 🚚  Fetching packages...
error @rezeus/korauter@1.0.0-beta.4: The engine "node" is incompatible with this module. Expected version "^7.6.0".
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

**Comment by iraycd** [comment]
Solved by using `yarn --ignore-engines add @rezeus/korauter`

**Comment by ozanmuyes** [comment]
Thanks for feedback, the problem is not because of `Buffer()` (interestingly enough the whole library doesn't use `Buffer` at all) - **but** the engine version. Node.js engine version entry fixed on the version [1.0.0-beta.5](https://github.com/rezeus/korauter/releases/tag/v1.0.0-beta.5).
